elements/chart/test/general.cy.js

```diff
@@ -23,4 +23,6 @@ describe("Chart", () => {
   it("loads the chart", () => loadChartTest());
 
   it("allows setting data values asynchronously", () => setDataValuesTest());
+
+  it("triggers an click.item event", () => clickChartTest());
```

Comment:
It should be tested that the event is not triggered when clicking outside of an item.

Reply:
Thanks, added in https://github.com/EOX-A/EOxElements/pull/1511/commits/2f2a01f012b06b52186a61e01e1e138cb8b8aa7e.